### PR TITLE
New serverless pattern - s3-lambda-transcribe-sam

### PR DESCRIPTION
**Pull request by anushreeumesh**
*Issue #, if available:* 1785

*Description of changes:*
This pattern contains a sample AWS Serverless Application Model (SAM) template that deploys a Lambda Function with an S3 _object created trigger_ to start an Amazon Transcribe job and place the results in another S3 bucket.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: s3-lambda-transcribe-sam/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "title": "S3 - Lambda - Transcribe",
+  "description": "Trigger a Transcribe job from an S3 upload.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This pattern contains a sample AWS Serverless Application Model (SAM) template that deploys a Lambda Function with an S3 object created trigger to start an Amazon Transcribe job and place the transcription results in another S3 bucket.",
```
Comment (bls20AWS) [inline s3-lambda-transcribe-sam/example-pattern.json:10]:
```suggestion
      "This pattern contains a Lambda Function with an S3 object trigger to start an Amazon Transcribe job and place the transcription results into a seperate S3 bucket.",
```

File: s3-lambda-transcribe-sam/example-pattern.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "This pattern deploys one Lambda Function and two S3 Buckets."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-lambda-transcribe-sam",
+      "templateURL": "serverless-patterns/s3-lambda-transcribe-sam",
+      "projectFolder": "s3-lambda-transcribe-sam",
+      "templateFile": "s3-lambda-transcribe-sam/template.yaml"
```
Comment (bls20AWS) [inline s3-lambda-transcribe-sam/example-pattern.json:19]:
```suggestion
      "templateFile": "template.yaml"
```

File: s3-lambda-transcribe-sam/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  "cleanup": {
+    "text": ["Delete the stack: <code>sam delete</code>."]
+  },
+  "authors": [
+    {
+      "name": "Anushree Umesh",
+      "image": "./umeshanu.jpeg",
+      "bio": "Anushree Umesh is an Associate Solutions Architect with Amazon Web Services",
+      "linkedin": "https://www.linkedin.com/in/anushreeumesh/"
```
Comment (bls20AWS) [inline s3-lambda-transcribe-sam/example-pattern.json:48]:
```suggestion
      "linkedin": "nushreeumesh"
```

File: s3-lambda-transcribe-sam/example-pattern.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  "testing": {
+    "text": ["See the GitHub repo for detailed testing instructions."]
+  },
+  "cleanup": {
+    "text": ["Delete the stack: <code>sam delete</code>."]
+  },
+  "authors": [
+    {
+      "name": "Anushree Umesh",
+      "image": "./umeshanu.jpeg",
```
Comment (bls20AWS) [inline s3-lambda-transcribe-sam/example-pattern.json:46]:
```suggestion
      "image": "/assets/images/resources/contributors/umeshanu.jpeg",
```

File: s3-lambda-transcribe-sam/example-pattern.json [hunk 5]
```diff
@@ -43,9 +43,9 @@
   "authors": [
     {
       "name": "Anushree Umesh",
-      "image": "./umeshanu.jpeg",
+      "image": "/assets/images/resources/contributors/umeshanu.jpeg",
       "bio": "Anushree Umesh is an Associate Solutions Architect with Amazon Web Services",
-      "linkedin": "https://www.linkedin.com/in/anushreeumesh/"
+      "linkedin": "nushreeumesh"
```
Comment (anushreeumesh) [inline s3-lambda-transcribe-sam/example-pattern.json:48]: "linkedin": "anushreeumesh"

**Pull request merged**